data-jdbc/src/test/groovy/io/micronaut/data/jdbc/h2/H2RepositorySpec.groovy

```diff
@@ -245,6 +248,15 @@ class H2RepositorySpec extends AbstractRepositorySpec implements H2TestPropertyP
         return true
     }
 
+    @PendingFeature
```

Comment:
Why is this `@PendingFeature`?

Reply:
Some combinations with `lower` are still broken and require more changes. Looks like I forgot to eliminate `PersistentPropertyBinaryPredicate`